InformationScripting/test/scripts/countChanges.py

```diff
@@ -0,0 +1,17 @@
+# script $<"<changes>" | "<countChanges>" | "<heatmap>">$
```

Comment:
You should definitely leave this script to serve an as example, but I think this particular query (the entire query) should be achievable without python. I guess we need to have a few common functions that do grouping/counting and this should be one of the use cases where we do that. This could be a future PR, but please have it in mind.